### PR TITLE
pomegranate & aphrodisiacs

### PR DESCRIPTION
**Pull request by SleepyScarecrow**
# Description

I've added in a new fruit and thee new reagents, this involved Pomegranates, and 3 aphrodisiac drugs. These have no actual effect other than OD punishment, and instead stand as an RP-based drug, a la the antidepressants in Delta-V.  I'm sure the ERPers among us will enjoy the addition.

---

# TODO

- [ ] Make Pomegranate more fleshed out; dishes, 
- [x] Additions

---

# Changelog

:cl: SleepyScarecrow
- add: Added pomegranates and three new drugs

### CODE REVIEW

File: Resources/Prototypes/Hydroponics/seeds.yml [hunk 1]
```diff
@@ -1032,6 +1032,33 @@
       Max: 20
       PotencyDivisor: 5
 
+- type: seed
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Hydroponics/seeds.yml:1035]:
```suggestion
- type: seed # Floofstation
```

File: Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/seeds.yml [hunk 2]
```diff
@@ -33,6 +33,7 @@
         - AloeSeeds
         - PoppySeeds
         - LingzhiSeeds
+        - PomegranateSeeds
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/seeds.yml:36]:
```suggestion
        - PomegranateSeeds # Floofstation
```

File: Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Mail/mail_civilian.yml [hunk 3]
```diff
@@ -57,6 +57,8 @@
       orGroup: Seeds
     - id: LingzhiSeeds
       orGroup: Seeds
+    - id: PomegranateSeeds
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Nyanotrasen/Entities/Objects/Specific/Mail/mail_civilian.yml:60]:
```suggestion
    - id: PomegranateSeeds # Floofstation
```

File: Resources/ServerInfo/Guidebook/Medical/Botanicals.xml
Comment (FoxxoTrystan) [inline Resources/ServerInfo/Guidebook/Medical/Botanicals.xml:1]: Why is this change here?
Just asking its not bad or good anyway.

File: Resources/Textures/Objects/Specific/Hydroponics/apple.rsi.zip
Comment (FoxxoTrystan) [inline Resources/Textures/Objects/Specific/Hydroponics/apple.rsi.zip:1]: Why is this here?

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
Overall good first PR, tho how would peoples know how to make the chems?
Mouth to ears or you want it listed on the documentation?

Good use of "Floofstation" folders, make sure to add comments on changes that are not in those folders.

Also i recomend to speak with "Lorelai Barker" on discord as he wanted to make Aphrodesiac, tho your current method seem the best way to do it honestly (please dont make it atmos gas)

**Review by Fansana — APPROVED** [review]

**Comment by SleepyScarecrow** [comment]
> Overall good first PR, tho how would peoples know how to make the chems? Mouth to ears or you want it listed on the documentation?
> 
> Good use of "Floofstation" folders, make sure to add comments on changes that are not in those folders.
> 
> Also i recomend to speak with "Lorelai Barker" on discord as he wanted to make Aphrodesiac, tho your current method seem the best way to do it honestly (please dont make it atmos gas)

Lorelai Inspired this, If they want gas they can use the foam mechanics. The documentation is already updated, at least in the guide itself, in the 'Chemicals' section.

**Comment by FoxxoTrystan** [comment]
> > Overall good first PR, tho how would peoples know how to make the chems? Mouth to ears or you want it listed on the documentation?
> > Good use of "Floofstation" folders, make sure to add comments on changes that are not in those folders.
> > Also i recomend to speak with "Lorelai Barker" on discord as he wanted to make Aphrodesiac, tho your current method seem the best way to do it honestly (please dont make it atmos gas)
> 
> Lorelai Inspired this, If they want gas they can use the foam mechanics. The documentation is already updated, at least in the guide itself, in the 'Chemicals' section.

Oh, guide auto-generate itself? good.
Take a look at changes needed please still.

File: Resources/Textures/Objects/Specific/Hydroponics/apple.rsi.zip
Comment (SleepyScarecrow) [reply Resources/Textures/Objects/Specific/Hydroponics/apple.rsi.zip:1]: Accidental, vscode just picked up on an accidental space when I was grabbing sprite bases.

File: Resources/ServerInfo/Guidebook/Medical/Botanicals.xml
Comment (SleepyScarecrow) [reply Resources/ServerInfo/Guidebook/Medical/Botanicals.xml:1]: Decided to count it as botanicals, as it is used to synthesize a drug.

**Comment by SleepyScarecrow** [comment]
> > > Overall good first PR, tho how would peoples know how to make the chems? Mouth to ears or you want it listed on the documentation?
> > > Good use of "Floofstation" folders, make sure to add comments on changes that are not in those folders.
> > > Also i recomend to speak with "Lorelai Barker" on discord as he wanted to make Aphrodesiac, tho your current method seem the best way to do it honestly (please dont make it atmos gas)
> > 
> > 
> > Lorelai Inspired this, If they want gas they can use the foam mechanics. The documentation is already updated, at least in the guide itself, in the 'Chemicals' section.
> 
> Oh, guide auto-generate itself? good. Take a look at changes needed please still.

Yes it did, and changes approved, thanks for pointing them out-- I'll try to tag those next time.

File: Resources/Textures/Objects/Specific/Hydroponics/apple.rsi.zip
Comment (FoxxoTrystan) [inline Resources/Textures/Objects/Specific/Hydroponics/apple.rsi.zip:1]: Again, why is this file here? i dont think this file should be here.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]

File: Resources/Textures/Objects/Specific/Hydroponics/apple.rsi.zip
Comment (SleepyScarecrow) [reply Resources/Textures/Objects/Specific/Hydroponics/apple.rsi.zip:1]: Oh- that's the zip. I'll remove the file when I get home & recommit. Apologies.

Comment (FoxxoTrystan) [reply Resources/Textures/Objects/Specific/Hydroponics/apple.rsi.zip:1]: Il remove it for you and run the tests.

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]
I will ask if you could move all the new textures and move it to the "Floof" directory.

"Resources/Textures/Floof"

This will make upsteam merge more easy and avoid overriding your changes.

File: Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml [hunk 4]
```diff
@@ -1822,3 +1822,32 @@
...
+    solutions:
+      food:
+        maxVol: 40
+        reagents:
+        - ReagentId: Pomelustine
+          Quantity: 30
+        - ReagentId: Nutriment
+          Quantity: 10
+  - type: Sprite
+    sprite: Objects/Specific/Hydroponics/pomegranate.rsi
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/produce.yml:1846]:
```suggestion
    sprite: Floof/Objects/Specific/Hydroponics/pomegranate.rsi
```

File: Resources/Prototypes/Entities/Objects/Specific/Hydroponics/seeds.yml [hunk 5]
```diff
@@ -373,6 +373,17 @@
...
+- type: entity # Floofstation
+  parent: SeedBase
+  name: packet of pomegranate seeds
+  description: "The Fruit of the Underworld"
+  id: PomegranateSeeds
+  components:
+    - type: Seed
+      seedId: pomegranate
+    - type: Sprite
+      sprite: Objects/Specific/Hydroponics/pomegranate.rsi
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Entities/Objects/Specific/Hydroponics/seeds.yml:385]:
```suggestion
      sprite: Floof/Objects/Specific/Hydroponics/pomegranate.rsi
```

File: Resources/Prototypes/Hydroponics/seeds.yml [hunk 6]
```diff
@@ -1032,6 +1032,33 @@
       Max: 20
       PotencyDivisor: 5
 
+- type: seed # Floofstation
+  id: pomegranate
+  name: seeds-pomegranate-name
+  noun: seeds-noun-seeds
+  displayName: seeds-pomegranate-display-name
+  plantRsi: Objects/Specific/Hydroponics/pomegranate.rsi
```
Comment (FoxxoTrystan) [inline Resources/Prototypes/Hydroponics/seeds.yml:1040]:
```suggestion
  plantRsi: Floof/Objects/Specific/Hydroponics/pomegranate.rsi
```

File: Resources/Textures/Floof/pomegranate.rsi/dead.png
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/pomegranate.rsi/dead.png:1]: Duplicate

File: Resources/Textures/Floof/pomegranate.rsi/harvest.png
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/pomegranate.rsi/harvest.png:1]: Duplicate

File: Resources/Textures/Floof/pomegranate.rsi/meta.json
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/pomegranate.rsi/meta.json:1]: Duplicate

File: Resources/Textures/Floof/pomegranate.rsi/produce.png
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/pomegranate.rsi/produce.png:1]: Duplicate

File: Resources/Textures/Floof/pomegranate.rsi/seed.png
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/pomegranate.rsi/seed.png:1]: Duplicate

File: Resources/Textures/Floof/pomegranate.rsi/stage-1.png
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/pomegranate.rsi/stage-1.png:1]: Duplicate

File: Resources/Textures/Floof/pomegranate.rsi/stage-2.png
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/pomegranate.rsi/stage-2.png:1]: Duplicate

File: Resources/Textures/Floof/pomegranate.rsi/stage-3.png
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/pomegranate.rsi/stage-3.png:1]: Duplicate

File: Resources/Textures/Floof/pomegranate.rsi/stage-4.png
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/pomegranate.rsi/stage-4.png:1]: Duplicate

File: Resources/Textures/Floof/pomegranate.rsi/stage-5.png
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/pomegranate.rsi/stage-5.png:1]: Duplicate

File: Resources/Textures/Floof/pomegranate.rsi/stage-6.png
Comment (FoxxoTrystan) [inline Resources/Textures/Floof/pomegranate.rsi/stage-6.png:1]: Duplicate

**Review by FoxxoTrystan — CHANGES_REQUESTED** [review]

**Review by FoxxoTrystan — APPROVED** [review]
